Source/Tests/ELFCodeLoader2.h

```diff
@@ -720,11 +730,18 @@ class ELFCodeLoader2 final : public FEXCore::CodeLoader {
     }
     else {
       // Nothing provided from the kernel, generate our own random values.
-      std::random_device rd;
-      std::uniform_int_distribution<uint64_t> d(0);
-
-      RandomLoc[0] = d(rd);
-      RandomLoc[1] = d(rd);
+      ssize_t Result{};
+      do {
+        // Use the getrandom syscall instead of std::random_device because std can assert.
```

Comment:
Moving this to a file-local helper template function allows us to avoid duplicating this docstring.

Reply:
Switched to local helper.